content/organization/intro/finance/index.en.md

```diff
@@ -61,6 +61,35 @@ Now as regards to the subject of the email, you want it to
 
 Refer to [The User Group Program](https://www.r-consortium.org/projects/r-user-group-support-program).
 
+R Consortium can offer grants to support the mission of local chapters in facilitating the exchange of knowledge and the formation of enduring R user group communities. These grants are aimed at assisting active user groups in their efforts to foster R-related learning and collaboration within their chapters. This document outlines the step-by-step process for applying for an R Consortium grant through R-Ladies.
+
+**Step 1: Express Interest**
+1. Contact [Batool Almarzouq](balmarzouq@turing.ac.uk) from the R-Ladies Global Team to express your interest in applying for an R Consortium grant.
+   - In your communication, specify the following details:
+     - The desired grant amount.
+     - The R-Ladies chapter for which you are applying.
+     - The name of the organizer(s) in your chapter who will be responsible for managing the grant.
+
+**Step 2: Complete the Online Application**
+3. Once Batool Almarzouq has received and recorded the above information, proceed to the grant application process. Access the grant application form [available on the R Consortium website](https://www.r-consortium.org/projects/r-user-group-support-program).
+   - Fill out all required fields accurately and comprehensively.
+   - Ensure that you have prepared a detailed proposal outlining the purpose and scope of your project.
+   - Submit the completed application form through the website.
+
```

Comment:
It looks like there is a "Step 3" missing, isn't it?

Reply:
I fixed this - thank you so much!